src/test/kotlin/org/mockito/kotlin/SuspendTest.kt

```diff
@@ -0,0 +1,37 @@
+package org.mockito.kotlin
+
+import kotlinx.coroutines.experimental.runBlocking
+import org.hamcrest.core.IsEqual
+import org.junit.Assert.assertThat
+import org.junit.Test
+import org.mockito.Mockito.*
+
+/**
+ * Test for mocking and verifying Kotlin suspending functions.
+ */
+class SuspendTest {
```

Comment:
I love the Kotlin test. We should have more of them I think!

Thanks a lot, this is our first ever official test coverage for different language :D